lib/npm-event.js

```diff
@@ -17,7 +17,8 @@ function npmEvent (server, {env, channel}, next) {
     config: {
       payload: {
         output: 'data',
-        parse: false
+        parse: false,
+        maxBytes: 26214400
```

Comment:
It is good practice to document what numbers mean. `maxBytes` is a good indicator of what we are looking at, but a small comment like `maxBytes: 26214400 // 24 * 1024 * 1024 = 24MB` can go a long way in longer-term maintainability. (hope this doesn’t sound too condescending ;D)